src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -141,4 +141,20 @@ public String getDisplayNameGiver() {
     public String getDisplayNameRecipient() {
         return displayNameRecipient;
     }
+    
+    public static List<FeedbackParticipantType> getParticipantListFromParams(String participantListParam) {
```

Comment:
earlier it was `participantListParam` but `participantListParams` makes more sense.Thanks for the tip.